src/Workspaces/Core/Portable/CodeActions/CodeActionOptions.cs

```diff
@@ -14,8 +14,21 @@ namespace Microsoft.CodeAnalysis.CodeActions
     internal readonly record struct CodeActionOptions(
         [property: DataMember(Order = 0)] SymbolSearchOptions SearchOptions,
         [property: DataMember(Order = 1)] bool HideAdvancedMembers = false,
-        [property: DataMember(Order = 2)] bool IsBlocking = false)
+        [property: DataMember(Order = 2)] bool IsBlocking = false,
+        [property: DataMember(Order = 3)] int WrappingColumn = CodeActionOptions.DefaultWrappingColumn)
```

Comment:
why is this in code action options instead of the wrapping options?

Reply:
WrappingColumn is not editor config option and so it must be read from global options - the CodeActionOptions contains such options.